src/config.ts

```diff
@@ -132,6 +132,11 @@ export interface RoutesConfig {
    */
   routesDir: string;
 
+  /**
+   * Routes filename; the filename of the generated route file ('routes.ts' by default)
+   */
+  routesFileName: string;
```

Comment:
This should be `routesFileName?: string;` since it's optional.

Reply:
I did this for typing reasons under the assumptions that all incoming settings would get their default value in `cli.ts` (which was very dumb, since I'm using TSOA programmatically too...)